src/Manager.php

```diff
@@ -664,6 +686,20 @@ private function userHasPermissionRecursive(
         return false;
     }
 
+    private function guestHasPermission(string $permissionName): bool
+    {
+        /** @psalm-suppress PossiblyNullArgument */
```

Comment:
```suggestion
```

Instead psalm suppress better create private method with annotation `@psalm-assert string $this->guestRole` and call him in `userHasPermission()` method.